src/features/PageLayout.tsx

```diff
@@ -23,7 +25,9 @@ export default function PageLayout({
         {nav}
       </PopUp>
       <div className="flex h-auto max-h-[calc(100vh-3.5rem)] min-h-[calc(100vh-3.5rem)] w-full md:max-h-[calc(100vh-4.25rem)] md:min-h-[calc(100vh-4.25rem)] md:pl-8 lg:pl-20 ">
-        <SideMenu className="hidden flex-shrink-0 overflow-y-auto pt-10 md:block">{nav}</SideMenu>
+        {cover.coverType === 1 && (
+          <SideMenu className="hidden flex-shrink-0 overflow-y-auto pt-10 md:block">{nav}</SideMenu>
```

Comment:
Frankly speaking, I don't like this approach to class declarations, it looks and reads better when classes are packed, but again, it's subjective. By packed I mean individual variables, without them this approach will only make it harder to read components when they start to growing.